packages/elements-core/src/utils/string.ts

```diff
@@ -1,3 +1,11 @@
 import { curry } from 'lodash';
 
 export const caseInsensitivelyEquals = curry((a: string, b: string) => a.toUpperCase() === b.toUpperCase());
+
+export function slugify(name: string) {
+  return name
+    .replace(/\/|{|}|\s/g, '-')
+    .replace(/-{2,}/, '-')
+    .replace(/^-/, '')
+    .replace(/-$/, '');
+}
```

Comment:
```suggestion
export function slugify(name: string) {
  return name
    .replace(/[\/{}\s-]+/g, '-')
    .replace(/^-|-$/g, '')
}
```
wouldn't this work?

Reply:
Not sure, I just moved this function - didn't write it originally. I'll check though.

Reply:
ah, I guess we have it in Studio, fair enough then.